xmodule/library_content_module.py

```diff
@@ -517,6 +523,22 @@ def refresh_children(self, request=None, suffix=None):  # lint-amnesty, pylint:
         self.tools.update_children(self, user_perms)
```

Comment:
Is this the only call to-re-update the course content to match the library content? or is  update_children called at times other than selection? This is for my own edification.

Reply:
In this PR state, it's used in the `studio_post_duplicate` method (line 571 of this file).
In the future, it will also be used for the library-sourced block